pkg/templates/basic.yml

```diff
@@ -0,0 +1,93 @@
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: NSTemplateTier
+metadata:
+  name: basic
+spec:
+  namespaces:
+    - type: ide
+      revision: abcdef
+      template: >
+        apiVersion: template.openshift.io/v1
+        kind: Template
+        metadata:
+          labels:
+            provider: codeready-toolchain
+            version: ${COMMIT}
+          name: basic-ide
+        objects:
+        - apiVersion: v1
+          kind: Namespace
+          metadata:
+            annotations:
+              openshift.io/description: ${PROJECT_NAME}-ide
+              openshift.io/display-name: ${PROJECT_NAME}-ide
+              openshift.io/requester: ${USER_NAME}
+            labels:
+              provider: codeready-toolchain
+              version: ${COMMIT}
+            name: ${PROJECT_NAME}
+        parameters:
+        - name: PROJECT_NAME
+          required: true
+        - name: USER_NAME
+          required: true
+        - name: COMMIT
+          required: true
```

Comment:
ok, let me replace `${PROJECT_NAME}` with `${USERNAME}` but let's address the `revision`/`commit` param in a subsequent since I'm going to refactor the manifests anyways

Reply:
done in fafe224, while extracting the templates from the NSTemplateTier manifests